core/API/DocumentationGenerator.php

```diff
@@ -48,9 +48,10 @@ public function __construct()
      *
      * @param bool $outputExampleUrls
      * @param string $prefixUrls
+     * @param bool $displayTitlesAsAngularDirective Set to false for the API ref doc at http://developer.piwik.org/api-reference/reporting-api where we need to display titles without using AngularJS
      * @return string
      */
-    public function getAllInterfaceString($outputExampleUrls = true, $prefixUrls = '')
+    public function getAllInterfaceString($outputExampleUrls = true, $prefixUrls = '', $displayTitlesAsAngularDirective = true)
```

Comment:
Instead of calling the API directly maybe it could be built a widget maybe? not sure